src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,203 @@
+<style scoped>@import url(/app/pages/interest-form/interest-form.css);</style>
+<h1>OpenStax College Interest Form</h1>
+<p>
+    Thank you for your interest in OpenStax College textbooks! Please complete the form below so that we may provide
+    you with the information you need to start saving your students thousands of dollars each semester.
+</p>
+<form accept-charset="UTF-8" action="/adoption_interests" class="new_adoption_interest" method="post">
+    <div style="margin:0;padding:0;display:inline">
+        <input name="utf8" type="hidden" value="✓">
+        <input name="authenticity_token" type="hidden" value="Xi14qRzCbSy2zkso1VEjKU4bEUfFWOrhWbQnh7tlO3c=">
```

Comment:
I don't think this should be a fixed value.  I'm assuming this is for CSRF.  It should be a random value provided by the server.  I believe Rails happens to call it an `authenticity_token`, although you could name it whatever makes sense.  You may need to talk to the backend though to see if they provide anything for this, whoever happens to be processing the form (is it SalesForce?).